charts/elasticsearch/values.yaml

```diff
@@ -71,6 +71,21 @@ resources: {}
   #   cpu: 100m
   #   memory: 128Mi
 
+# How long to wait for elasticsearch to stop gracefully
+terminationGracePeriod: 120
+
+sysctlVmMaxMapCount: 262144
+
+readinessProbe:
+  failureThreshold: 3
+  initialDelaySeconds: 30
+  periodSeconds: 10
+  successThreshold: 3
+  timeoutSeconds: 5
+
+# https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-health.html#request-params wait_for_status
+clusterHealthCheckParams: "wait_for_status=yellow&timeout=10s"
+
```

Comment:
It is not enough to add such values here. You need to modify the helm template for elasticsearch and add these values inside the lms repo